addons/website/static/src/client_actions/configurator/configurator.js

```diff
@@ -277,6 +281,29 @@ class PaletteSelectionScreen extends Component {
         this.state.selectPalette(paletteName);
         this.props.navigate(ROUTES.featuresSelectionScreen);
     }
+
+    /**
+     * Gets the attachment id of the current logo.
+     *
+     * @private
+     * @returns {Promise<number>} the attachment id
+     */
+    async getLogoAttachmentId() {
+        const websiteId = (await this.orm.call("website", "get_current_website")).match(/\d+/)[0];
+        const [attachment] = await this.orm.searchRead("ir.attachment", [
+            ["name", "=", "logo"],
+            ["website_id", "=", parseInt(websiteId)],
+        ], ["id"], { limit: 1 });
```

Comment:
A website can only be configured once. See `configurator_done`. Then in the configurator the user can:
1. Upload a logo if he didn't already do it
2. Replace an existing logo (that he already uploaded) => We remove the previous one
3. Delete the uploaded logo => We remove the previous one.

The only tricky case I see is the following:
- Create a new website
- Via the URL, leave the configurator (go to '/')
- In edit mode upload a logo and save
- Now install a new module (or go back manually to /website/configurator)
=> You can upload a new logo throw the configurator.

Note that this flow is very tricky and I'm not sure what is the best to do in this case.
1. If the user uploads a logo we remove all previously added logo (but then if he changes his mind and remove the logo, he lost his first logo)
2. Always remove **one** previous logo so that the user would keep its first logo in any case

I don't have a strong opinion on this. I let you choose :)

Reply:
Hmm indeed it really seems to be a corner case (like no one will do that in reality) :thinking: 

Okay let's not care about potential multiple previous logo, seems to have more risk in removing them (as discussed IRL) :slightly_smiling_face: